packages/components/src/core/ButtonIcon/index.stories.tsx

```diff
@@ -93,245 +94,362 @@ export const Default = {
+          aria-label="xMark"
+          style={{ marginRight: spacings?.m }}
+          sdsIcon="xMark"
+          sdsSize="medium"
+          sdsType="tertiary"
+        />
+      </div>
+      <div style={{ display: "flex" }}>
+        <ButtonIcon
+          aria-label="barChartVertical3"
+          style={{ marginRight: spacings?.s }}
+          sdsIcon="barChartVertical3"
+          sdsSize="small"
+          sdsType="primary"
+        />
+        <ButtonIcon
+          aria-label="barChartVertical3"
+          style={{ marginRight: spacings?.s }}
+          sdsIcon="barChartVertical3"
+          sdsSize="small"
+          sdsType="primary"
+        />
+      </div>
+      <div style={{ display: "flex" }}>
+        <ButtonIcon
+          aria-label="plusCircle"
+          style={{ marginRight: spacings?.s }}
+          sdsIcon="plusCircle"
+          sdsSize="small"
+          sdsType="secondary"
+        />
+        <ButtonIcon
+          aria-label="plusCircle"
+          style={{ marginRight: spacings?.s }}
+          sdsIcon="plusCircle"
+          sdsSize="small"
+          sdsType="secondary"
+        />
+      </div>
+      <div>
+        <ButtonIcon
+          aria-label="xMark"
+          style={{ marginRight: spacings?.s }}
+          sdsIcon="xMark"
+          sdsSize="small"
+          sdsType="tertiary"
+        />
+      </div>
+    </div>
+  );
 };
 
 export const LivePreview = {
   parameters: {
-    controls: {
-      exclude: ["disabled", "on", "sdsIcon", "sdsSize", "sdsType"],
-    },
     snapshot: {
       skip: true,
     },
   },
+  render: (args: Args) => <LivePreviewDemo {...args} />,
+};
+
+// Screenshot test
+
+const ScreenshotTestDemo = (): JSX.Element => {
+  const topLevel: React.CSSProperties = {
+    columnGap: "20px",
+    display: "inline-grid",
+    fontFamily: "sans-serif",
+    marginRight: "50px",
+  };
+  const displayContents: React.CSSProperties = {
+    display: "contents",
+  };
+  const penultimateLevel: React.CSSProperties = {
+    display: "contents",
+  };
+  const bottomLevel: React.CSSProperties = {
+    marginBottom: 10,
+  };
+  const fontWeightNormal: React.CSSProperties = {
+    fontWeight: "normal",
+  };
+  const topLabel: React.CSSProperties = {
+    ...fontWeightNormal,
+    fontSize: "2em",
+    gridColumn: "1 / 6",
+    marginBottom: 0,
+  };
+  const midLabel: React.CSSProperties = {
+    ...fontWeightNormal,
+    borderStyle: "solid none none none",
+    gridColumn: "1 / 6",
+    justifySelf: "stretch",
+    paddingTop: 10,
+  };
+  const secondLabel: React.CSSProperties = {
+    ...midLabel,
+    borderWidth: "2px",
+    fontSize: "1.17em",
+    margin: "20px 0",
+  };
+  const thirdLabel: React.CSSProperties = {
+    ...midLabel,
+    alignSelf: "end",
+    borderWidth: "1px",
+    fontWeight: "normal",
+    margin: "0 0 5px 0",
+  };
+  const bottomLabel: React.CSSProperties = {
+    ...fontWeightNormal,
+    margin: "10px 0",
+  };
 
-  render: (props: Args): JSX.Element => {
```

Comment:
This goes down lower in the code now, becuase I switched it to first build a "...Demo" component which is then called by the main component lower down (line 445).